app/src/test/java/com/codesoom/assignment/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,123 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import com.codesoom.assignment.errors.NotSupportedIdException;
+import io.jsonwebtoken.Claims;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("JwtUtil 테스트")
+class JwtUtilTest {
+    private static final String SECRET = "12345678901234567890123456789010";
+
+    private JwtUtil jwtUtil;
+
+    @BeforeEach
+    void setUp() {
+        jwtUtil = new JwtUtil(SECRET);
+    }
+
+    @Nested
+    @DisplayName("encode 메서드")
+    class Describe_encode {
+
+        @Nested
+        @DisplayName("유효한 id이 주어지면")
```

Comment:
유효한 id의 종류는 어떤 것이 있나요? 만약 null 이 아니기만 해도 유효하다면 한시적으로 테스트 코드에서 설명해 주어도 괜찮습니다.

```suggestion
        @DisplayName("유효한 id가 주어지면")
```

Reply:
한시적으로 테스트 코드에서 설명해 주어도 된다는 말이 문구로 설명해 주면 되나요?

`null이 아닌 유효한 id가 주어지면` 이런 식으로 작성해 주면 될까요?

Reply:
`null`은 대단히 특별한 지위를 갖고 있어서 테스트에 직접적으로 언급해 주는 것이 바람직한 경우도 많습니다.